app/javascript/packs/session-timeout-ping.js

```diff
@@ -84,6 +86,15 @@ function success(data) {
       timeRemaining,
       timeTimeout,
     );
+    if (srCountdownInterval) {
+      clearInterval(srCountdownInterval);
+    }
+    srCountdownInterval = login.srCountdownTimer(
+      document.getElementById('sr-countdown'),
+      timeRemaining,
+      timeTimeout,
+      20 * 1000,
```

Comment:
where does the `20` come from?

Reply:
20 seconds was the time recommended to me to wait in between message updates for the screen reader. Should I list that in a comment here or turn it into an env variable in the yml file?

Reply:
> 20 seconds was the time recommended to me to wait in between message updates for the screen reader. Should I list that in a comment here or turn it into an env variable in the yml file?

It makes sense to me as a constant, which we have some precedent to define toward the top of the file ([example](https://github.com/18F/identity-idp/blob/7ad3feab24f6f9e0e45224d9e9be9458c0a6a648/app/javascript/packages/rails-i18n-webpack-plugin/extract-keys-webpack-plugin.js#L3-L17)).

i.e.

```js
const SR_MESSAGE_UPDATE_INTERVAL_SECONDS = 20;
```